build/ci/richnav.yml

```diff
@@ -0,0 +1,51 @@
+# Name: DotNet-Project-System Rich Code Navigation
+# URL: https://dev.azure.com/devdiv/DevDiv/_build?definitionId=9675
+#
+# Creates the Rich Navigation index for navigation throughout the master branch
+# of dotnet/project-system
+#
+
+# Branches that trigger a build on commit
+trigger:
+  branches:
+    include:
+    - master
+    - dev*
+    - feature/*
+
+# Branches that trigger builds on PR
+pr:
+  branches:
+    include:
+    - master
+    - dev*
+    - feature/*
+
+resources:
+- repo: self
+  clean: true
+queue:
+  name: VSEng-MicroBuildVS2019
+  demands: Cmd
+  timeoutInMinutes: 15
+variables:
+  BuildConfiguration: Debug
+  BuildPlatform: any cpu
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+  EnableRichCodeNavigation: true
+  
+steps:
+- task: NuGetRestore@1
+  displayName: Restore 'Microsoft.DotNet.IBCMerge' package
+  inputs:
+    solution: 'build\proj\internal\RestoreIBCMerge.csproj'
+    feed: '8f470c7e-ac49-4afe-a6ee-cf784e438b93'
+
+- script: $(Build.SourcesDirectory)\build.cmd /build /ci /no-deploy /no-integration /ibc /no-clearnugetcache /configuration $(BuildConfiguration)
+  displayName: Build ProjectSystem.sln
+
+- task: RichCodeNavIndexer@0
+  displayName: RichCodeNav Upload
+  inputs:
+    languages: 'csharp'
```

Comment:
Just to have a better understanding, we don't need to explicitly specify VB here?

Reply:
Nope! When you pass csharp it indexes both csharp and VB